From 2f19c5e76d84e2d91a133de0c8c9415973e906ed Mon Sep 17 00:00:00 2001
From: danglin <danglin@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Sat, 6 Sep 2014 21:53:14 +0000
Subject: [PATCH 227/326] 	PR testsuite/56194 	*
 g++.dg/init/const9.C: Skip scan-assembler-not "rodata" on hppa*-*-*.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_8-branch@214998 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/testsuite/ChangeLog            | 5 +++++
 gcc/testsuite/g++.dg/init/const9.C | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/gcc/testsuite/ChangeLog b/gcc/testsuite/ChangeLog
index d8f884a..0c91c6d 100644
--- a/gcc/testsuite/ChangeLog
+++ b/gcc/testsuite/ChangeLog
@@ -1,3 +1,8 @@
+2014-09-06  John David Anglin  <danglin@gcc.gnu.org>
+
+	PR testsuite/56194
+	* g++.dg/init/const9.C: Skip scan-assembler-not "rodata" on hppa*-*-*.
+
 2014-09-03  Marek Polacek  <polacek@redhat.com>
 
 	Backport from mainline
diff --git a/gcc/testsuite/g++.dg/init/const9.C b/gcc/testsuite/g++.dg/init/const9.C
index ba1dfd4..d733e95 100644
--- a/gcc/testsuite/g++.dg/init/const9.C
+++ b/gcc/testsuite/g++.dg/init/const9.C
@@ -1,5 +1,5 @@
 // PR c++/55893
-// { dg-final { scan-assembler-not "rodata" } }
+// { dg-final { scan-assembler-not "rodata" { target { ! hppa*-*-* } } } }
 
 struct foo
 {
-- 
2.0.3

